backend/app/api.py

```diff
@@ -8,15 +8,22 @@
 import requests
 from . import crud, models, schemas
 from .database import SessionLocal, engine
-from .routers import chatgpt, auth
+from .routers import chatgpt
+
+import os
+from supabase import create_client, Client
+from .authHandler import JWTBearer
 
 load_dotenv()
 models.Base.metadata.create_all(bind=engine)
 
+url: str = os.environ.get("SUPABASE_URL")
+key: str = os.environ.get("SUPABASE_KEY")
+supa: Client = create_client(url, key)
```

Comment:
are url, Key, and supa, required here?

Reply:
No, we don't need these for JWT connect, I leave here now, to see if we need for future(data structure or user information).
In env (backend), we can leave supabase url and key as default.

Reply:
ok we can leave for now.  But we should determine how Maggie's postgres setup relates to supabase version of postgres .

Reply:
Yes agree, for now, Maggie's setup for pg is all about testing, which is not related to this project. I will remove these files later. 

 About auth route, I think we need auth route in backend. I removed it, because the original auth route imported from the `auth.py` which we talked about removing that one. But we need a new auth route in the backend.